src/brevitas/nn/quant_avg_pool.py

```diff
@@ -38,6 +39,9 @@ def __init__(
         AvgPool2d.__init__(self, kernel_size=kernel_size, stride=stride)
         QuantLayerMixin.__init__(self, return_quant_tensor)
         TruncMixin.__init__(self, trunc_quant=trunc_quant, **kwargs)
+        self.cache_inference_quant_act = False
+        self.cache_quant_io_metadata_only = True
+        self.cache_class = None
```

Comment:
Worth considering whether this should be its own class? Or should some of these be attributes of `_CachedIO`?

Any views on that?